backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -26,6 +26,29 @@ include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+1]
 include::modules/oadp-using-data-mover-for-csi-snapshots.adoc[leveloffset=+1]
 
+[id="oadp-cleaning-up-after-data-mover-1-1-backup"]
+== Cleaning up after a backup using Data Mover with OADP 1.1.
+
+For OADP 1.1., you must perform a data cleanup after you perform a backup using any version of Data Mover.
+
+The cleanup comprises deleting:
```

Comment:
```suggestion
The cleanup comprises of deleting:
```

Reply:
@kaovilai Actually, the IBM Style Guide says not to use "comprises," with or without "of."  I changed the sentence to "The cleanup consists of deleting the following resources:"